src/components/SpeechBubble.jsx

```diff
@@ -0,0 +1,39 @@
+import styled from "styled-components";
+
+const BubbleContainer = styled.div`
```

Comment:
So fun to see that you used styled components! But i see that you combines it with CSS, which one do you prefer? :)